.circleci/test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -ex
+
+cd /tmp/pytorch/xla
+
+export XLA_USE_XRT=1 XRT_DEVICE_MAP="CPU:0;/job:localservice/replica:0/task:0/device:XLA_CPU:0"
+export XRT_WORKERS="localservice:0;grpc://localhost:40934"
+
+python test/test_operations.py
+python test/test_train_mnist.py --tidy
+./test/cpp/run_tests.sh
```

Comment:
You need to run the script from `./test/cpp`, the script doesn't run correctly otherwise:

```./test/cpp/run_tests.sh
/tmp/pytorch/xla/build /tmp/pytorch/xla
CMake Error: The source directory "/tmp/pytorch/xla" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.
make: *** No targets specified and no makefile found.  Stop.
/tmp/pytorch/xla
```

That being said, why didn't we get a failure because of it? Doesn't `run_tests.sh` return a failure status? We should make sure that failed tests show up as such.

Reply:
@asuhan Nice catch! The failure didn't show up because we don't have `set -e` in `run_tests.sh` so that it doesn't stop when error happens. And the last command in the script succeeded so that hide the error. Should be good after the fix commit. :)